docs/sql-ref-syntax-qry-select-lateral-view.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: global
+title: LATERAL VIEW Clause
+displayTitle: LATERAL VIEW Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+`LATERAL VIEW` clause is used in conjunction with UDTF such as explode(), which will generate a vitual table containing one or more rows. `LATERAL VIEW` will  apply the rows to each original output rows.
```

Comment:
vitual -> virtual?

Reply:
`explode` is a UDTF? How about "in conjunction with generator functions such as `explode`, ..."?

Reply:
remove a single space: `will  apply` -> `will apply`

Reply:
`each original output rows.` -> `each original output row.`